charts/kube-state-metrics/templates/podsecuritypolicy.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.podSecurityPolicy.enabled }}
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
This could be a single line:

```
{{- if and .Values.podSecurityPolicy.enabled .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```